### PR TITLE
docs: Add help center changes for the new presences feature

### PR DESCRIPTION
**Pull request by mErrenst**
Closes https://github.com/famedly/product-management/issues/1541

### CODE REVIEW

File: i18n/de/docusaurus-plugin-content-docs/current/help-center/using-famedly/chats.mdx [hunk 1]
```diff
@@ -194,6 +194,18 @@ Das Laden des Archivs kann einige Zeit in Anspruch nehmen.
   </TabItem>
 </Tabs>
 
+## Den Online-Status anderer Nutzer prüfen
+
+Sie können den Online-Status anderer Nutzer auf den Avataren in der Raumliste, der Kontaktliste (beim Erstellen eines neuen Chats), in der Titelleiste von 1:1 Chats sowie in Gruppenchats in den einzelnen Nachrichten und der Mitgliederliste sehen.
```
Comment (rvanwaveren) [inline i18n/de/docusaurus-plugin-content-docs/current/help-center/using-famedly/chats.mdx:199]: This I do not understand. How can I see the online status in the single messages within a group chat?

File: i18n/de/docusaurus-plugin-content-docs/current/help-center/using-famedly/chats.mdx [hunk 2]
```diff
@@ -194,6 +194,18 @@ Das Laden des Archivs kann einige Zeit in Anspruch nehmen.
   </TabItem>
 </Tabs>
 
+## Den Online-Status anderer Nutzer prüfen
+
+Sie können den Online-Status anderer Nutzer auf den Avataren in der Raumliste, der Kontaktliste (beim Erstellen eines neuen Chats), in der Titelleiste von 1:1 Chats sowie in Gruppenchats in den einzelnen Nachrichten und der Mitgliederliste sehen.  
+
+Welches Symbol zeigt welchen Online-Status?
+
+<b>Online</b> &rarr; Grüner Punkt auf dem Avatar
```
Comment (rvanwaveren) [inline i18n/de/docusaurus-plugin-content-docs/current/help-center/using-famedly/chats.mdx:203]: Please check this, I believe in the help center we do not use the term Avatar anywhere. It is the profile picture or the initials.

**Review by rvanwaveren — COMMENTED** [review]
@mErrenst Keine große Einwände, nur eine Sache die ich nicht ganz verstanden habe und einen Punkt zum Wortwahl.

File: i18n/de/docusaurus-plugin-content-docs/current/help-center/using-famedly/chats.mdx (hunk 1 above)
Comment (mErrenst) [reply i18n/de/docusaurus-plugin-content-docs/current/help-center/using-famedly/chats.mdx:199]: In group chats every message other parties write have their avatar showed above. The avatar has a presence indicator: 
image

**Pull request merged**